Dockerfile

```diff
@@ -1,20 +1,22 @@
-FROM centos:7
+FROM quay.io/centos/centos:stream8
 
 RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
-    yum -y install epel-release git && \
-    yum -y install python2-pip && \
+    yum -y install git glibc-langpack-en epel-release && \
+    yum -y install python3-pip && \
```

Comment:
You could fold this into the previous line, saving one yum transaction. But not tooo important.